tests/SyntaxTest.php

```diff
@@ -57,4 +57,42 @@ public function testBuilder()
         $this->assertSame('==', $syntax->getParameterValueSeparator());
         $this->assertSame('""', $syntax->getParameterValueDelimiter());
         }
+
+    /**
+     * @dataProvider getDataForSpacesTrimmedCorrectly
+     */
+    public function testSpacesTrimmedCorrectly($incoming, $expected)
+        {
+            $propertiesToTest = array(
+
+                'OpeningTag',
+                'ClosingTag',
+                'ClosingTagMarker',
+                'ParameterValueSeparator',
+                'ParameterValueDelimiter'
+            );
+
+            foreach ($propertiesToTest as $property) {
+
+                $getter = "get$property";
+                $setter = "set$property";
+
+                $builder = new SyntaxBuilder();
+                $syntax = $builder->$setter($incoming)->getSyntax();
+                $this->assertEquals($expected, $syntax->$getter());
```

Comment:
If at all please call those methods by hand, variable variables method calling in such a simple case is not a good idea.